### ISSUE TITLE
new Formeo() object doesn't work

### ISSUE THREAD

**Issue by NomanAhmed6352** [Draggable/formeo#259]
"var formeo = new Formeo() " is not working

i am using following script:
    <script src="https://draggable.github.io/formeo/assets/js/formeo.min.js"></script>

**Comment by buseodaci** [comment]
@nomi89 Did u solve that ? I'm also new and didn't get that.

**Comment by NomanAhmed6352** [comment]
No, I remembered i changed strategy and applied some alternate solution. Tell me your objective hope it will be of some help.

**Comment by buseodaci** [comment]
I want to use Formeo in my Angular Project. I get **formeo.min.js** and **formeo.min.css** from **formeo** which under the node_modules. In docs they say i use this code but its give many errors.
![image](https://user-images.githubusercontent.com/25960852/85827255-9c13be00-b78e-11ea-9ed2-d5ce883292a9.png)
I found  [demo.min.js](https://github.com/pankaj-git/formeo_angular/blob/master/src/assets/js/demo.js) and called in my project and i dont know how to control  and edit project.
